graph.go

```diff
@@ -418,3 +418,12 @@ func (g *Graph) EdgesMap() map[string][]Edge {
 func (g *Graph) HasNode(n Node) bool {
 	return g == n.graph
 }
+
+// GetAttributes returns the attributes
```

Comment:
better have ".. returns a copy if the attributes." 

So the user knows it cannot modify the map directly